### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by Duretitibesso3**
Portfolio setup and mobile version skeleton project
In this project I:

Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git
Added index.html, style.css, and test.md file
Created a mobile version skeleton for my portfolio
Add work section and About My self section.
Added README file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,7 +37,144 @@ <h3 id="foot">LET’S CONNECT</h3>
       </ul>
     </div>
   </section>
-</div>
-<script src="https://kit.fontawesome.com/3bf83b5817.js" crossorigin="anonymous"></script>
-</body>
-</html>
+    </div>
+      <!--Work section-->
+    <div class="Work section">
```
Comment (danielufeli) [inline index.html:42]: - [ ] Great working on you work section, kindly Add some `padding` between the tags and the button so it aligns with the Figma Template.

Figma Template
pad-2

## Your Design
pad-1

File: index.html [hunk 2]
```diff
@@ -37,7 +37,144 @@ <h3 id="foot">LET’S CONNECT</h3>
...
+        </ul>
+        <div>
+          <button type='button'>See Project</button>
+        </div>
+      </section>
+    </div>
+    
+    <!--About me-->
+    <div>
+      <section class="about">
```
Comment (danielufeli) [inline index.html:127]: - [ ] Great working on you about section, kindly Add some `padding` between the social icons and button so they align with the Figma Template.

- [ ] Kindly work on the button design to look like the Figma Template

## Figma Template
pad-4

## Your Design
pad-3

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Duretitibesso3, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

a

## Highlights
Nice commits messages✔️
Descriptive PR ✔️
Clean Design ✔️
Good use of Github flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -37,7 +37,144 @@ <h3 id="foot">LET’S CONNECT</h3>
...
+        <h3 id="connect">LET'S CONNECT </h3>
+        <div class="soc">
+          <i class="fa-brands fa-twitter"></i>
+        <i class="fa-brands fa-linkedin-in"></i>
+        <i class="fa-duotone fa-m"></i>
+        <i class="fa-brands fa-github"></i>
+        <i class="fa-brands fa-angellist"></i>
+        </div>
+        <img src="images/social-media-icons.png" alt="" aria-hidden="true">
+        <button type='button'>Get my resume</button>
```
Comment (zainsadaqat) [inline index.html:140]: ## As asked by the previous Code Reviewer, Please work on the button and make it look similar to the Figma Design. 👍🏻 
### Your Design
Screen Shot 2022-04-29 at 5 47 40 PM

### Required Design
Screen Shot 2022-04-29 at 5 48 02 PM

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁 
# Hello  @Duretitibesso3  😃 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zhadier — APPROVED** [review]
## Hi @Duretitibesso3 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

# Optional Changes
- [ ] Your links don't seem to have padding between them as you can see in the image below and they have a different color compared to the Figma template. kindly do try to fix it before you merge :)

image


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**